docs/design/resources.md

```diff
@@ -101,8 +101,9 @@ following:
 1. A user edits the `EtcdCluster` resource to increase the size by
    one.
 2. Using the logic above, the cluster operator notes that the cluster
-   is undersized by one and adds a new peer to the etcd cluster.
-3. The cluster operator then notices that the cluster expects a peer
+   is undersized by one and adds a new member to the etcd cluster
+   using the etcd API.
+3. The cluster operator then notices that the cluster expects a member
```

Comment:
Perhaps best to say `the EtcdCluster controller` instead of `cluster operator`? I'd define the 'operator' as the 'manager' binary

Reply:
Clarified in 73c531b